### PR TITLE
1322 - Ship logs to Logstash 

### PR DESCRIPTION
**Pull request by tavy88**
### Context

Ship logs to Logstash

### Changes proposed in this pull request

Add the rails semantic logger needed to format logs
and have them displayed as one line inside Logit dashboard.

Screenshot 2023-03-22 at 17 11 20
Screenshot 2023-03-22 at 18 50 25

### Link to Trello card

https://trello.com/c/lMeq5bya

### Checklist

- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: spec/logs/formatters/log_stash_formatter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,116 @@
+require "rails_helper"
+
+# rubocop:disable RSpec/AnyInstance
```
Comment (gpeng) [inline spec/logs/formatters/log_stash_formatter_spec.rb:3]: Can we don this in .rubocop.yml instead so they're all in one place?

**Review by gpeng — COMMENTED** [review]
This looks good to me. @saliceti is there anything else we need to do for this?

File: spec/logs/formatters/log_stash_formatter_spec.rb (hunk 1 above)
Comment (tavy88) [reply spec/logs/formatters/log_stash_formatter_spec.rb:3]: I didn't want to fully disable it for everything just for this exceptional case.

Comment (tavy88) [reply spec/logs/formatters/log_stash_formatter_spec.rb]: @gpeng updated!

**Pull request merged**